src/js/main.js

```diff
@@ -9,3 +9,17 @@ jQuery.ajax('../apis/github/users/octocat.json').then(function(data){
  console.log(arguments);
 });
 })();
+
```

Comment:
This looks a lot like the jQuery found here: http://inspirationalpixels.com/tutorials/creating-tabs-with-html-css-and-jquery#step-jquery

Always attribute your source!! And be sure you modify the code appropriately to work with your current project.